### DISCUSSION TITLE
How to place HasMany field into a Tab

### DISCUSSION THREAD

**Discussion by benjy8001** [moonshine-software/moonshine#1312] [Q&A] [answered]
Hi,

I try to place a HasMany field into a Tab, like this :
```
                            Tab::make(__('moonshine::ui.resource.stocks'), [
                                HasMany::make(__('moonshine::ui.resource.models.label'), relationName: 'carModels', resource: new CarModelResource)
                                    ->async()
                                    ->creatable()
                                    ->hideOnDetail()
                                    ->hideOnIndex(),
                            ]),
```

But the HasMany is always placed at the bottom of the form : 
![image](https://github.com/user-attachments/assets/bacc43ce-c1d5-48f6-9aaa-a3b66a2f27f9)

How can i place this HasMany into a Tab ?

Thanks for your help !

**Comment by benjy8001** [comment]
I found this article https://cutcode.dev/articles/kastomizaciia-stranicy-formy-moonshine-20 .

So i figure out how to "move" this HasMany into other tab :
![image](https://github.com/user-attachments/assets/fbeba65e-77ec-4ed8-80e9-1e310264a2e0)

But, in the other hand, i loose my field split into different Tab. I also want to move logo part (simple Url field) in a new tab :
![image](https://github.com/user-attachments/assets/e382af18-3305-47cb-9bab-41451db75ef0)

Like this : 
![image](https://github.com/user-attachments/assets/7e4ad65e-fd77-471f-9d95-4fd063d75411)


How can i do this ?

**Comment by lee-to** [comment, marked as answer]
https://cutcode.dev/articles/kastomizaciia-stranicy-formy-moonshine-20

**Reply by benjy8001** [reply]
Thanks, i just follow the article, it's working, but i try to split some form parts into different tabs, but i don't know how to do it.
